app/controllers/messages/highlights_controller.rb

```diff
@@ -12,6 +12,7 @@ def update
     private
 
     def set_message
+      # TODO: scope by organization
```

Comment:
I'm not sure about this one, finding a message by its id should not have any way to leak from an organization to another. On the other hand, if we assign the message correctly to the organization, we shouldn't have an issue finding it within the organizations messges.

Reply:
better to scope it as if the users messes with the urls/requests messages from other organizations could be highlighted

Reply:
done with: https://github.com/tactilenews/100eyes/pull/1996